packages/paste-core/components/spinner/src/styled.tsx

```diff
@@ -1,23 +1,31 @@
 import { styled } from "@twilio-paste/styling-library";
 
 import { circleCircumference } from "./constants";
-import { CircleKeyframes, SvgKeyframes } from "./keyframes";
+import { CircleKeyframes, SafariSvgKeyframes, SvgKeyframes } from "./keyframes";
 
 export const StyledCircleTrack = styled.circle({
   transformOrigin: "center",
   opacity: 0.25,
 });
 
-export const AnimatedStyledCircle = styled.circle<{ show: boolean }>(({ show }) => ({
-  transformOrigin: "center",
-  animation: `1.5s ease-in-out infinite both ${CircleKeyframes}`,
-  strokeDasharray: circleCircumference,
-  opacity: show ? 1 : 0,
-}));
+export const AnimatedStyledCircle = styled.circle<{ show: boolean }>(({ show }) => {
+  // `transformOrigin: "center"` does not work on Safari, so only apply the animation on non-Safari browsers
+  const isSafari = navigator.userAgent.includes("Safari") && !navigator.userAgent.includes("Chrome");
```

Comment:
This value should be set into state in a useEffect. Because on SSR navigator is undefined.

Reply:
We should pass it into this and StyledSVG as a prop

Reply:
done and done